Source/Charts/Components/AxisBase.swift

```diff
@@ -200,15 +192,31 @@ open class AxisBase: ComponentBase
     /// Flag indicating that the axis-max value has been customized
     internal var _customAxisMax: Bool = false
     
-    /// Do not touch this directly, instead, use axisMinimum.
-    /// This is automatically calculated to represent the real min value,
-    /// and is used when calculating the effective minimum.
-    internal var _axisMinimum = Double(0)
+    /// The minimum value for this axis.
+    /// If set, this value will not be calculated automatically depending on the provided data.
+    /// Use `resetCustomAxisMin()` to undo this.
+    @objc open internal(set) var axisMinimum = 0.0 {
+        willSet {
+            _customAxisMax = true
+        }
+
+        didSet {
+            axisRange = abs(axisMaximum - axisMinimum)
+        }
+    }
     
-    /// Do not touch this directly, instead, use axisMaximum.
-    /// This is automatically calculated to represent the real max value,
-    /// and is used when calculating the effective maximum.
-    internal var _axisMaximum = Double(0)
+    /// The maximum value for this axis.
+    /// If set, this value will not be calculated automatically depending on the provided data.
+    /// Use `resetCustomAxisMax()` to undo this.
+    @objc open internal(set) var axisMaximum = 0.0 {
```

Comment:
Same to axisMinimum. Seems you missed some context to make setter internal?

Reply:
I think I was just tired and missed a couple things ; )

Reply:
Using Demo to test could help trigger the errors?